apps/vaporgui/RenderHolder.cpp

```diff
@@ -224,6 +227,45 @@ void RenderHolder::_initializeNewRendererDialog(vector<string> datasetNames) {
 	}
 }
 
+void RenderHolder::_showIntelDriverWarning(const string &rendererType)
+{
+    if (GLManager::GetVendor() != GLManager::Vendor::Intel)
+        return;
+    if (rendererType != VolumeRenderer::GetClassType() &&
+        rendererType != VolumeIsoRenderer::GetClassType())
+        return;
+    
+    ParamsMgr *paramsMgr = _controlExec->GetParamsMgr();
+    SettingsParams *sp = (SettingsParams *)paramsMgr->GetParams(SettingsParams::GetClassType());
+    if (sp->GetDontShowIntelDriverWarning())
+        return;
+    
+    // Qt will automatically delete this for us apparently
```

Comment:
I don't think this is the case. Scott has spent some time researching into QT memory management; you might want to ask him.